### ISSUE TITLE
Licenses for Embed for your Org scenario to be adjusted

### ISSUE THREAD

**Issue by tarekbanims** [MicrosoftDocs/powerbi-docs#3118]
For the licensing requirements, Embed for your org should be Pro, PPU, or P SKU.
![image](https://user-images.githubusercontent.com/85665574/121472581-6f25de80-c9d2-11eb-9b99-98ed7bbf0292.png)
the Article mentions Pro, PPU and A SKU 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 90078026-b608-ccf2-f3ef-58021b484f75
* Version Independent ID: e51ab568-5610-ff18-c5c0-4d2f27935dfa
* Content: [Embed content in your Power BI embedded analytics application enabling better embedded BI insights for your organization - Power BI](https://docs.microsoft.com/en-us/power-bi/developer/embedded/embed-sample-for-your-organization?tabs=net-core)
* Content Source: [powerbi-docs/developer/embedded/embed-sample-for-your-organization.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/master/powerbi-docs/developer/embedded/embed-sample-for-your-organization.md)
* Service: **powerbi**
* Sub-service: **powerbi-developer**
* GitHub Login: @KesemSharabi
* Microsoft Alias: **kesharab**

**Comment by kfollis** [comment]
@tarekbanims - thank you for the feedback. I'm assigning this to the writer to verify the licensing requirements and update as needed.

**Comment by KesemSharabi** [comment]
Hi @tarekbanims,

The note is correct, however it is misleading. The third bullet states that you need 'a capacity' - not an A (Azure) capacity. I'll rephrase it to make clearer.

Thanks for getting involved and for pointing this out!

Kesem

**Comment by KesemSharabi** [comment]
Hi @tarekbanims,

I reworded the note so that it's no longer confusing (see [Prerequisites](https://docs.microsoft.com/power-bi/developer/embedded/embed-sample-for-your-organization?tabs=net-core#prerequisites)).

![image](https://user-images.githubusercontent.com/54834915/122894958-59100a80-d350-11eb-9bc3-63a888962449.png)

Thanks,

Kesem
